test/15_renderSubForm/test.js

```diff
@@ -130,39 +206,46 @@ define(['BlinkForms', 'testUtils', 'BIC'], function (Forms, testUtils) {
           done();
       });
 
-      test('subform with subform', function (done) {
+      test('subforms with further subform', function (done) {
           var result;
-          var xml = "<Address><Address><id>51</id><Detail>Great work</Detail><Exp><Exp><id>1</id><Rank>45</Rank></Exp></Exp></Address><Address><id>52</id><Detail>Bad news</Detail><Exp><Exp><id>2</id><Rank>89</Rank></Exp><Exp><id>3</id><Rank>88</Rank></Exp></Exp></Address><Address><id>53</id><Detail>Quite Day</Detail><Exp/></Address></Address>";
+          var xml = "<subform><Address><id>51</id><Detail>Great work</Detail><Description><Exp><id>1</id><Rank>45</Rank><Status/></Exp></Description></Address><Address><id>52</id><Detail>Bad news</Detail><Description><Exp><id>2</id><Rank>89</Rank><Status><Status><Age>45</Age></Status></Status></Exp><Exp><id>3</id><Rank>88</Rank><Status/></Exp></Description></Address><Address><id>53</id><Detail>Quite Day</Detail><Description/></Address></subform>";
           var record = {
-            Address: [
+            subform: [
               {
                 id: '51',
                 Detail: 'Great work',
-                Exp: [
+                Description: [
                   {
                     id: '1',
-                    Rank: '45'
+                    Rank: '45',
+                    Status: ''
                   }
                 ]
               },
               {
                 id: '52',
                 Detail: 'Bad news',
-                Exp: [
+                Description: [
```

Comment:
It didn't. It's just git being dumb. Just the right number of lines were removed above to make it confused about 2 similar looking sections of code. The diff wouldn't be doing this if I'd done the un-nesting as a separate commit.